src/ring_theory/adjoin_root.lean

```diff
@@ -96,13 +96,13 @@ noncomputable instance field : discrete_field (adjoin_root f) :=
 ideal.quotient.field (span {f} : ideal (polynomial α))
 
  -- short-circuit type class inference
-instance : is_field_hom (coe : α → adjoin_root f) := by apply_instance
+instance : is_ring_hom (coe : α → adjoin_root f) := by apply_instance
```

Comment:
Travis is complaining about another `adjoin_root.is_ring_hom`, looks like it's at line 47.

Reply:
Hmm, weird that those weren't caught by the global search-replace... I'll fix this tomorrow.

Reply:
I misunderstood the issue. Removed the duplicate instances.